<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>flight_chaninfo</class>
 <widget class="QWidget" name="flight_chaninfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>911</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QLineEdit 样式 */
QLineEdit {
    color: #101011;
    background-color: white;
    font-size: 24px; /* 提高字体大小 */
    border: 1px solid #7E7F7F; /* 设置边框颜色和粗细 */
    border-radius: 10px; /* 圆角边框 */
    font-family: &quot;Microsoft YaHei&quot;;
    padding: 5px; /* 增加内边距 */
}
</string>
  </property>
  <widget class="QLineEdit" name="ecoCapacity">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>670</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrivalAirport">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>410</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="ecoPrice">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>560</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>690</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>座位数</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="deparureAirport">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>410</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>航班号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="busPrice">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrivalCity">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>330</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>是否延误</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="busCapacity">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>670</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>340</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>目的地</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="arrivalTime">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="date">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>220</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>420</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>到达机场</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>130</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>航空公司</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>230</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>出发时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>580</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="headCapacity">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>670</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>出发机场</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 28px;
    color: white;
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                  stop:0 #FDC245, stop:1 #FB9E23);
border-radius: 10px;

}
QPushButton:hover {
background-color:#FAC85C;
}</string>
   </property>
   <property name="text">
    <string>修改</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>130</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>机型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>出发地</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="aircraftModel">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>120</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="airCompany">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>120</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="headPrice">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>560</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
border: 2px solid #BDBEC1; /* 边框 */
	border-top: 6px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="deparureCity">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>330</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>日期</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>500</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>商务舱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QListView" name="listView">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>911</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">/* QLineEdit 样式 */
QLineEdit {
    color: #101011;
    background-color: white;
    font-size: 24px; /* 提高字体大小 */
    border: 1px solid #7E7F7F; /* 设置边框颜色和粗细 */
    border-radius: 10px; /* 圆角边框 */
    font-family: &quot;Microsoft YaHei&quot;;
    padding: 5px; /* 增加内边距 */
}
</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>到达时间</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="isDelayed">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>340</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>经济舱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_23">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>500</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Microsoft YaHei&quot;;
    font-size: 24px;
    color: #101011;
	background-color: #FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>头等舱</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="deparureTime">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>220</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="flightNumber">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>120</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;SimSun&quot;;
    font-size: 22px;
    color: #101011;
	background-color: #FFFFFF;
	border-bottom: 2px solid #7E7F7F;

}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>listView</zorder>
  <zorder>ecoCapacity</zorder>
  <zorder>arrivalAirport</zorder>
  <zorder>ecoPrice</zorder>
  <zorder>label_25</zorder>
  <zorder>deparureAirport</zorder>
  <zorder>label_14</zorder>
  <zorder>busPrice</zorder>
  <zorder>arrivalCity</zorder>
  <zorder>label_18</zorder>
  <zorder>busCapacity</zorder>
  <zorder>label_16</zorder>
  <zorder>arrivalTime</zorder>
  <zorder>date</zorder>
  <zorder>label_19</zorder>
  <zorder>label_13</zorder>
  <zorder>label_10</zorder>
  <zorder>label_24</zorder>
  <zorder>headCapacity</zorder>
  <zorder>label_20</zorder>
  <zorder>add</zorder>
  <zorder>label_12</zorder>
  <zorder>label_17</zorder>
  <zorder>aircraftModel</zorder>
  <zorder>airCompany</zorder>
  <zorder>headPrice</zorder>
  <zorder>deparureCity</zorder>
  <zorder>label_11</zorder>
  <zorder>label_22</zorder>
  <zorder>label_9</zorder>
  <zorder>isDelayed</zorder>
  <zorder>label_21</zorder>
  <zorder>label_23</zorder>
  <zorder>deparureTime</zorder>
  <zorder>flightNumber</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
